src/routes/Tool/Competences.tsx

```diff
@@ -15,6 +16,35 @@ import {
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
+const hardCodedOpportunities = [
```

Comment:
mockOpportunities could be better variable name.

Reply:
Yeah, that is better. Changed to suggested name.